package.json

```diff
@@ -99,7 +99,7 @@
     "postcss-discard-duplicates": "^4.0.2",
     "postcss-inline-svg": "^3.1.1",
     "postcss-loader": "^2.1.4",
-    "prettier": "^1.12.1",
+    "prettier": "^2.0.5",
```

Comment:
Required to support optional chaining. It was technically added in 1.19.0 but I figured why not 🤷‍♂️ . Someone tell me if you disagree.

Reply:
Ended up just using 1.19.1